tsc/frr/sonic_frr_update_process.md

```diff
@@ -0,0 +1,79 @@
+# **SONiC Community FRR Upgrade Process Proposal**
+
+FRR upgrade and patch cadence is largely on need basis in the current SONiC release program. This SONiC FRR upgrade process proposal is to formalize the FRR upgrade cadence and process for future SONiC release programs.  
+
+# SONiC FRR Version Upgrade History
+
+<p align=center>
+<img src="frr.png" alt="">
+</p>
+
+
+# FRR Project Release Cadence
+-   FRR release numbering scheme x.y.z-s#
+-   New FRR releases roughly every 4 months
+	- Apr 2023 - Release 8.5.1 (48 fixes)
+	- Mar 2023 - Release 8.5 (947+ commits)
+	- Jan 2023 - Release 8.4.2 (22 fixes)
+	- Nov 2022 - Release 8.4.1 (16 fixes)
+	- Nov 2022 - Release 8.4 (700+ commits)
+	- Aug 2022 - Release 8.3.1 (14 fixes)
+	- Jul 2022 - Release 8.3 (1000+ commits)
+	- Mar 2022 - Release 8.2.2 (800+ commits)
+	- Nov 2021 - Release 8.1.0 (1200+ commits)
+	- Jul 2021 - Release 8.0.0 (2200+ commits)
+
+-   SONiC to stay out from major/minor releases (x.y) and use patch release (.z) for stability (eg, FRR 8.3.1 instead of 8.3 if it is for 202211 release)
```

Comment:
What's the guideline to go for major/minor FRR releases for SONIC community? only the features drive that decision? or we'll always be 2 or 3 releases behind to get some features soak cycles (to get better stability) at the FRR community?

Reply:
The recommendation is to go with the latest .z release, eg, currently in frrouting.org/release there are 9.0.1 (9/5/23), 8.5.3 (9/4/23), 9.0 (8/4/23), etc, if we are to upgrade now, the guidance is to go with 9.0.1.

Reply:
OK, for example, let's say 9.0.1 not released, what we have is 9.0 and 8.5.3 releases to choose, which one do we choose here,?I think, we choose 8.5.3 instead of 9.0, is my understanding correct? please clarify this part in the document.

Reply:
yes correct, updated in new revision